e2e/typescript_3.1/BUILD.bazel

```diff
@@ -42,3 +55,37 @@ jasmine_node_test(
         ":test_lib",
     ],
 )
+
+nodejs_binary(
+    name = "main_js",
+    data = [
+        ":main",
+    ],
+    entry_point = "e2e_typescript_3_1/main.js",
+)
+
+nodejs_binary(
+    name = "main_js_sm",
+    data = [
+        ":main",
+        "@npm//source-map-support",
+    ],
+    entry_point = "e2e_typescript_3_1/main.js",
+)
+
+rollup_bundle(
+    name = "bundle",
+    entry_point = "main",
+    deps = [
+        ":main",
+    ],
+)
+
+rollup_bundle(
```

Comment:
Should change the `test` script in package.json to `bazel build ... && bazel test ...` so that these targets are built

Reply:
I am pretty sure `bazel test ...` implicitly also builds all targets (unless `--build_tests_only`) is also specified. Let me verify this though.

Reply:
Yep just verified. I had a failing `rollup_bundle` and `bazel test ...` failed on building it.

Reply:
Ahhhh. Good to know. Thanks for verifying.